BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -259,37 +325,27 @@ namespace big
 
 		if (ImGui::Checkbox("Headlight##headlight_en", &xenon))
 		{
-			g_fiber_pool->queue_job([] {
-				VEHICLE::TOGGLE_VEHICLE_MOD(player_vehicle, MOD_XENON_LIGHTS, xenon);
-			});
+			VEHICLE::TOGGLE_VEHICLE_MOD(player_vehicle, MOD_XENON_LIGHTS, xenon);
 		}
 		ImGui::SameLine();
 		if (ImGui::Checkbox("Left", &neon_left))
 		{
-			g_fiber_pool->queue_job([] {
-				VEHICLE::SET_VEHICLE_NEON_LIGHT_ENABLED_(player_vehicle, NEON_LEFT, neon_left);
-			});
+			VEHICLE::SET_VEHICLE_NEON_LIGHT_ENABLED_(player_vehicle, NEON_LEFT, neon_left);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.